packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
+  display: flex;
+  flex-direction: row;
+  color: var(--grey100);
+  padding-top: 32px;
+  padding-bottom: 32px;
+  container-name: CdgAbout;
+  container-type: inline-size;
+  justify-content: space-between;
+
+  .cdg-about-left {
+    width: 50%;
+  }
+
+  .cdg-about-right {
+    background: var(--accent);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+}
+
+@media screen and (max-width: 960px) {
```

Comment:
Bring these responsive selector to responsive file, for this case 960 most likely medium size, tablet screen size. So you should move to _medium.scss file. We also have _small.scss file.

Reply:
fixed